django/db/models/aggregates.py

```diff
@@ -31,8 +35,9 @@ def input_field(self):
 
     @property
     def default_alias(self):
-        if hasattr(self.source_expressions[0], 'name'):
-            return '%s__%s' % (self.source_expressions[0].name, self.name.lower())
+        expressions = self.get_source_expressions()
+        if len(expressions) == 1 and hasattr(expressions[0], 'name'):
```

Comment:
We're not interested in supporting that use case for default_alias. We've discussed this on a previous PR before, but default_alias isn't something we want to keep adding to.

```
Model.objects.annotate(GroupConcat('field_a', 'field_b'))  # group_concat__field_a__field_b
Model.objects.annotate(merged_name=GroupConcat('field_a', 'field_b'))
```

I don't see much value in trying to construct a complex default alias when it's extremely simple for users to give the result a name that makes sense in the context of their query.